l10n_it_fatturapa_out_di/wizard/wizard_export_fatturapa.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Marco Colombo - Phi srl
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+
+from odoo import models
+from odoo.addons.l10n_it_account.tools.account_tools import encode_for_export
+from odoo.addons.l10n_it_fatturapa.bindings.fatturapa import AltriDatiGestionaliType
+
+class WizardExportFatturapa(models.TransientModel):
+    _inherit = "wizard.export.fatturapa"
+
+    def setDettaglioLinea( self, line_no, line, body, price_precision, uom_precision):
+        DettaglioLinea = super().setDettaglioLinea(line_no, line, body, price_precision, uom_precision)
+
+        if line.force_dichiarazione_intento_id:
```

Comment:
Le dichiarazione d'intento di norma sono presenti nel [campo](https://github.com/OCA/l10n-italy/blob/1574ff1d64fe06eb418cefa622f982d03640f499/l10n_it_dichiarazione_intento/models/account_invoice.py#L13) e non vengono quindi esposte.

Il campo usato qui serve se si ha necessità di applicare parzialmente una o più dichiarazioni a quanto mi risulta.

Reply:
AltriDatiGestionali è comunque per riga. Come suggerisci di procedere?   (questa domanda riguarda anche il modulo per la 14 ha la stessa logica)

Va messo secondo te l'elemento <AltriDatiGestionali> per tutte le righe fattura quando la DI è presente in testata fattura?

Reply:
Come da commento sotto, non sembra obbligatorio il collegamento riga -> DI, per cui secondo me si può generalizzare con una riga di commento in cui inserire tutte le dichiarazioni presenti, sia nel campo generale che nelle righe.